front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -22,22 +23,27 @@ export const MODERATOR_ROUTES = [
   '/admin/insights',
 ];
 
-export const isModeratorRoute = (item: IRouteItem) => {
-  return MODERATOR_ROUTES.includes(item.path);
+export const isModeratorIndexRoute = (item: IRouteItem) => {
+  return MODERATOR_INDEX_ROUTES.includes(item.path);
 };
 
 export const isModeratedProjectRoute = (
   item: IRouteItem,
   user: IUser | null
 ) => {
-  const idRegexp = /^\/admin\/projects\/([a-z0-9-]+)\//;
-  const matches = idRegexp.exec(item.path);
+  const { pathname: pathNameWithoutLocale } = removeLocale(item.path);
+  const idRegexp = /^\/admin\/projects\/([a-z0-9-]+)\/?/;
```

Comment:
And the ? here fixes the issue with the trailing slash for project editing, as you proposed in your PR.